<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminview</class>
 <widget class="QMainWindow" name="adminview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>608</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>874</width>
    <height>608</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>874</width>
    <height>608</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>管理员终端</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>811</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="scores_tab">
     <attribute name="title">
      <string>成绩查询</string>
     </attribute>
     <widget class="QTableView" name="score_table">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>771</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="add_score_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>增添成绩</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>581</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>按照(哪一个项目)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="item_combo">
         <item>
          <property name="text">
           <string>学号</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>综合成绩</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>考试平均成绩</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>任课老师评分</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>同学互评</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>品德思想</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>语文</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>数学</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>英语</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>进行(什么样的排序)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="order_comBox">
         <item>
          <property name="text">
           <string>升序</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>降序</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>440</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>删除成绩</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_score">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>440</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>修改成绩</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>学生查询</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>581</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>学号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="id_search_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>只读(不允许修改任何内容)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>731</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTableView" name="studenttableView">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>681</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>721</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Outline</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="help_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
